charts/artifact-hub/templates/db_migrator_job.yaml

```diff
@@ -10,9 +10,18 @@ metadata:
     "helm.sh/hook-weight": "0"
     "helm.sh/hook-delete-policy": before-hook-creation
 {{- end }}
+{{- if .Values.dbMigrator.job.extraJobLabels }}
```

Comment:
Can you switch to `with` blocks instead of `if`? They are a bit easier to maintain (no need to repeat the path) and it'll help with consistency as we use it across most templates.